Updating to sprig 3.2.0

mattfarina opened the pull request:
Note, randInt is now a function in sprig so the failing test needed
to be updated to a function that does not exist.

jdolitsky reviewed (changes requested):
@mattfarina - can you resolve conflicts?

mattfarina committed:
Updating to sprig 3.2.0

Note, randInt is now a function in sprig so the failing test needed
to be updated to a function that does not exist.

Signed-off-by: Matt Farina <matt@mattfarina.com>

mattfarina commented:
@jdolitsky conflicts have been fixed.
